web/modules/custom/block_exchange/src/Plugin/Block/ExchangeBlock.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Drupal\block_exchange\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+
+/**
+ * Provides a 'Hello' Block.
+ *
+ * @Block(
+ *   id = "block_exchange",
+ *   admin_label = @Translation("Exchange Block")
+ * )
+ */
+class ExchangeBlock extends BlockBase {
+
+  /**
+   * {@inheritdoc}
+   */
+
+  public function build()
+  {
+
+    $j = @file_get_contents('https://api.privatbank.ua/p24api/pubinfo?json&exchange&coursid=5');
```

Comment:
Variable name should describe what is inside.
In this case, you have the response from API.